tox.ini

```diff
@@ -69,8 +69,8 @@ basepython = python3
 envdir = {toxworkdir}/docs
 commands =
     python housekeep.py prep
-    pytest -v aiosmtpd/docs
-    python setup.py build_sphinx
+    sphinx-build --color -b doctest -d build/.doctree aiosmtpd/docs build/doctest
+    sphinx-build --color -b html    -d build/.doctree aiosmtpd/docs build/html
```

Comment:
This makes things a bit faster, actually, since the doctest builder and the html builder shared the same doctree.